cypress/integration/questions.ts

```diff
@@ -0,0 +1,65 @@
+import faker from 'faker';
+
+context('Questions tests', () => {
+  before(() => {
+    cy.resetDB();
+  });
+
+  beforeEach(() => {
+    cy.viewport(1100, 1100);
+    cy.visit('/');
+  });
+
+  const textQuestion = faker.lorem.words(2);
+
+  it('User officer search questions', () => {
+    cy.login('officer');
+
+    cy.navigateToTemplatesSubmenu('Proposal templates');
+
+    cy.contains('default template')
+      .parent()
+      .get("[title='Edit']")
+      .first()
```

Comment:
I don't think we need `.first()` here because if I am correct you are getting the template by name and then parent element should have only one edit.